en/setup/getting_started.md

```diff
@@ -4,11 +4,77 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
 
 ## Step 1: Install Mautic
 
-If you have already downloaded the zip from the download page or have installed Mautic through some other source (Softaculous, Bitnami, Digital Ocean etc...) then you have already completed the first step. If not then you will need to upload the Mautic package (a zip file) to your server; unzip the files; and then navigate to that location in your browser. You will find Mautic has a very easy to follow on-screen installation process.
+If you have already downloaded the zip from the download page or have installed Mautic through some other source (Softaculous, Bitnami, Digital Ocean etc...) then you have already completed the first step. If not then you will need to upload the Mautic package (a zip file) to your server; unzip the files; and then navigate to that location in your browser. 
+
+As an example, if your domain is mydomain.com and you have unzipped the file into a folder called 'm', you would then browse to mydomain.com/m to start the installation process.
+
+###Permission errors
+If you see errors when you try to access the installation as below:
+
+![](/setup/media/permissions-error.png)
+
+the problem is likely to be due to the permissions on your files and folders.  
+
+If you have access to the command line, you can run the following commands to reset the file and folder permissions.  Make sure that you are in the directory where Mautic has been installed (using the example above, that would mean executing these commands within /m in your web root folder - you can get there using the `cd` command)
+
+```find . -type d -exec chmod 755 {} \;
```

Comment:
I believe the 3 starting and ending backticks must be on a single line so the Markdown parser would style the content in between as a code sample.

What's wrong with this PR, Ruth? What shouldn't be part of this PR?

Reply:
![screenshot-github com-2019 09 02-10_07_41](https://user-images.githubusercontent.com/2930593/64103251-b5fc9400-cd69-11e9-80f5-9b3b76f508f9.png) I see this on the PR which I don't think should be there (it was me bringing my fork up to date with the upstream repo).

In regards the backticks I was following https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code so I guess I need to add a line break after the final line?

Reply:
Yes, there should be a line break.

I don't think the merge commit harms anything. I refresh the master branch to ensure it is exactly the same as the remote one like this:
`$ git fetch upstream`
`$ git reset --hard upstream/master`
⚠️ but it will delete all changes and commits that are not in the remote master branch.

Reply:
You are right that the merge commit shouldn't harm anything.  However it is nicer to avoid it if possible since it does make the history more complicated to look at.  I provided a suggestion below for using rebase to get rid of the merge.